req.py

```diff
@@ -98,12 +102,17 @@ def generate_account(driver, rand_num):
 
     time.sleep(1.5)
 
-    print(f"successfully made account for fake email {email}")
+    print(f"successfully made account for fake email {fake_identity['email']}")
 
-def fill_out_application_and_submit(driver, rand_num):
+def fill_out_application_and_submit(driver, rand_num, fake_identity={'first_name': fake.first_name(), 'last_name': fake.last_name(), 'email': fake.free_email()}):
 
     driver.implicitly_wait(10)
     city = list(cities.keys())[rand_num]
+
+    # make resume
+    make_resume(fake_identity['first_name']+' '+fake_identity['last_name'], fake_identity['email'], 'resume.pdf')

```

Comment:
Referencing t[his comment](https://github.com/SeanDaBlack/KelloggBot/pull/17#issuecomment-991388174). For the 3rd parameter here, you could pass in a randomly generated resume file name

Reply:
Added!